src/ucm/util/reloc.h

```diff
@@ -51,8 +51,8 @@ ucs_status_t ucm_reloc_modify(ucm_reloc_patch_t* patch);
  *
  * @return Original function pointer for 'symbol'.
  */
-static void* UCS_F_MAYBE_UNUSED
-ucm_reloc_get_orig(const char *symbol, void *replacement)
+static UCS_F_MAYBE_UNUSED
+void* ucm_reloc_get_orig(const char *symbol, void *replacement)
```

Comment:
seems need to fix also these, similarly:
ucs_ptr_array_dump
uct_rc_mlx5_devx_create_cmd_qp
uct_rc_mlx5_verbs_create_cmd_qp

Reply:
Done. Force push with small(<50) LOC change

Reply:
Also `uct_ib_mlx5_iface_fill_attr`

Reply:
@gleon99 Done. Sorry for missing it.